src/modules/LanItems.rb

```diff
@@ -1193,7 +1201,8 @@ def Read
     # 3) variables which keeps (some of) attributes of the current item (= item
     # which is being pointed by the iterator)
     def reset_cache
-      LanItems.Items = {}
+# FIXME: what is suitable replacement with new config storage?
```

Comment:
probably something like `Yast::Config.yast = Yast::Config.system`

Reply:
this would reset everything (even e.g. Routing). And hidden question is "Find out whether it is still needed"